paparazzi/src/main/java/app/cash/paparazzi/SnapshotVerifier.kt

```diff
@@ -71,6 +72,7 @@ public class SnapshotVerifier @JvmOverloads constructor(
           """.trimIndent()
         )
         ImageUtils.assertImageSimilar(
+          relativePathFailure = expectedFailure.path,
```

Comment:
as a byproduct of the work in https://github.com/cashapp/paparazzi/issues/1465#issuecomment-2214901484, I'd like to avoid passing more state into this function.